config_mitie.json

```diff
@@ -1,5 +1,5 @@
 {
-  "backend": "mitie",
+  "backend": "mitie_sklearn",
```

Comment:
I think so. Technically this backend is meant to be faster, so we should consier defaulting it. 
@amn41 what do you think?